api/src/main/java/org/openmrs/module/locationbasedaccess/LocationUtils.java

```diff
@@ -31,4 +31,13 @@ public static Location getPersonLocation(Person person) {
         }
         return null;
     }
+
+    public static Boolean doesPersonBelongToGivenLocation(Person person, PersonAttributeType personAttributeType, String sessionLocationUuid) {
+        PersonAttribute personAttribute = person.getAttribute(personAttributeType);
+        return (personAttribute != null && compare(personAttribute.getValue(), sessionLocationUuid));
+    }
+
+    public static Boolean compare(String value1, String value2) {
```

Comment:
Didn't we already have the compare method somewhere else?

Reply:
Nope, I have just moved from the PatientSearchAdvicer class to LocationUtils calls (I need this for the PersonSearchAdvicer class also).